test/e2e/set/scyllacluster/scyllacluster_replace.go

```diff
@@ -42,7 +42,14 @@ var _ = g.Describe("ScyllaCluster replace", func() {
 		sc, err = waitForScyllaClusterState(waitCtx1, f.ScyllaClient().ScyllaV1(), sc.Namespace, sc.Name, scyllaClusterRolledOut)
 		o.Expect(err).NotTo(o.HaveOccurred())
 
-		verifyScyllaCluster(ctx, f.KubeClient(), sc)
+		di, err := NewDataInserter(ctx, f.KubeClient().CoreV1(), sc, getMemberCount(sc))
```

Comment:
let's be consistent by either using a fixed number or the getMemberCount function in all the places

Reply:
We only use a fixed number in scyllacluster_scaling, because inserting data with RF=2, scaling down and then reading with consistency ALL is impossible. Setting it to `1` explicitly in that one case seems fine to me. All the other test cases use `getMemberCount`.